jdk/src/solaris/bin/java_md_solinux.c

```diff
@@ -634,7 +657,7 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                     /* On AIX we additionally need 'jli' in the path because ld doesn't support $ORIGIN. */
                     JLI_StrLen(jrepath) + JLI_StrLen(arch) + JLI_StrLen("/lib//jli:") +
                     /* On AIX P9 or newer with NX accelerator enabled, add the accelerated zlibNX to LIBPATH */
```

Comment:
While you're here, can you finish the sentence with a period?

Reply:
Updated for all versions and created https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/669